### PR TITLE
Added check_assumptions in Main Namespace

### PR DESCRIPTION
**Pull request by arihantparsoya**
This is Fix for #11505. Added `check_assumptions()` in main namespace by adding it in _init_.py.
#### Sample Program:

``` python
>>> from sympy import *
>>> check_assumptions(-5, integer=True)
True
```

### CODE REVIEW

**Comment by parsoyaarihant** [comment]
@asmeurer , can you review this?

File: sympy/solvers/solvers.py [hunk 1]
```diff
@@ -331,6 +331,9 @@ def check_assumptions(expr, **assumptions):
        >>> z = Symbol('z')
        >>> check_assumptions(z, real=True)
     """
+    if e is not None:
```
Comment (asmeurer) [inline sympy/solvers/solvers.py]: Can you document this option in the docstring?

Comment (asmeurer) [inline sympy/solvers/solvers.py]: Maybe a better name for this argument would be `against` (i.e., you are checking the assumptions of `expr` *against* the assumptions of `against`).

**Comment by parsoyaarihant** [comment]
@asmeurer , I have updated the docstrings.

File: sympy/solvers/solvers.py [hunk 2]
```diff
@@ -325,12 +324,21 @@ def check_assumptions(expr, **assumptions):
        >>> check_assumptions(-2*x - 5, real=True, positive=True)
        False
 
+       To check assumptions of `expr` against another variable or expression,
+       pass the expression or variabls as `against`.
```
Comment (asmeurer) [inline sympy/solvers/solvers.py]: Spelling

Comment (parsoyaarihant) [inline sympy/solvers/solvers.py]: Thanks, required changes are committed.

File: sympy/solvers/solvers.py [hunk 3]
```diff
@@ -325,12 +324,21 @@ def check_assumptions(expr, **assumptions):
        >>> check_assumptions(-2*x - 5, real=True, positive=True)
        False
 
+       To check assumptions of `expr` against another variable or expression,
+       pass the expression or variable as `against`.
```
Comment (asmeurer) [inline sympy/solvers/solvers.py]: Use two backquotes in docstrings.

Comment (parsoyaarihant) [inline sympy/solvers/solvers.py]: Thanks, changes are committed.

**Pull request merged**